### PR TITLE
Fix messages background for the latest Slack version

### PR DESCRIPTION
**Pull request by tectiv3**
I had to remove `@-moz-document regexp("https://[^./]*\\.slack\\.com/(?!pricing)(?!security).*") {}`  in built css for styles to work inside Slack app.

### CODE REVIEW

**Comment by sandfox** [comment]
Just tested this branch and it looks to be working on my desktop OSX installation

**Comment by jkremser** [comment]
same here, works well on my Fedora and desktop version (Electron) of slack :shipit:

File: README.md [hunk 1]
```diff
@@ -9,6 +9,33 @@ Unfortunately, if you're using the desktop version of Slack, you will not** be a
 
 ** However you can still use this theme using this little hack: https://github.com/laCour/slack-night-mode/issues/73#issuecomment-242707078
 
+#### Instructions for macOS
+- Open `/Applications/Slack.app/Contents/Resources/app.asar.unpacked/src/static/ssb-interop.js`
+- Add this snippet to the bottom of that file
+- Replace `/absolute/path/to` with path to your `black.css`
+- You will have to re-add this everytime Slack update itself.
```
Comment (rhoboat) [inline README.md]: Reads better as:
`You will have to re-add this every time Slack updates itself.`

**Comment by laCour** [comment]
Thanks for your efforts @tectiv3! I thought I might explain why another pull request was preferred in place of yours.

The CSS files for these themes are compiled, and should not be edited directly. SCSS files are used instead, and you can learn more about the process in the [CONTRIBUTING.md](https://github.com/laCour/slack-night-mode/blob/master/CONTRIBUTING.md) file.

**Comment by tectiv3** [comment]
Thank you for taking time to respond.
I've read contributing guide, I've applied changes to SCSS files but I also uploaded generated css. My mistake was to remove regexp from css/black.css instead of using css/raw.
Anyway, how do you feel about including desktop client modifications instructions in the readme (also part of this PR)?

**Pull request closed without merging**